Documentation/ceph-cluster-crd.md

```diff
@@ -353,17 +397,15 @@ spec:
 ```
 
 * Ensure that `master` matches the network interface of the host that you want to use.
-* The NAD should be referenced along with the namespace in which it is present like `public: <namespace>/<name of NAD>`.
-  e.g., the network attachment definition are in `rook-multus` namespace:
 * Ipam type `whereabouts` is required because it makes sure that all the pods get a unique IP address from the multus network.
-
-```yaml
-  public: rook-multus/rook-public-nw
-  cluster: rook-multus/rook-cluster-nw
-```
-
-This is required in order to use the NAD across namespaces.
-* In Openshift, to use the NetworkAttachmentDefinition across namespaces, the NAD must be deployed in the default namespace and it can be referenced as `default/myNAD` where `default` is the namespace and `myNAD` is the network attachment definition.
+* The NAD should be referenced along with the namespace in which it is present like `public: <namespace>/<name of NAD>`.
+  e.g., the network attachment definition are in `default` namespace:
+  ```yaml
+    public: default/rook-public-nw
+    cluster: default/rook-cluster-nw
+  ```
+  * This format is required in order to use the NAD across namespaces.
+  * In Openshift, to use the NetworkAttachmentDefinition across namespaces, the NAD must be deployed in the default namespace and it can be referenced as `default/myNAD` where `default` is the namespace and `myNAD` is the network attachment definition.
```

Comment:
Let's rewrite this to be more clear.
In Openshift, to use a NetworkAttachmentDefinition (NAD) across namespaces, the NAD must be deployed in the `default` namespace. The NAD is then referenced with the namespace: `default/rook-public-nw`

Reply:
Made the changes. Thanks